awx/ui_next/src/components/DetailList/Detail.jsx

```diff
@@ -25,24 +25,29 @@ const DetailValue = styled(({ fullWidth, ...props }) => (
   `}
 `;
 
-const Detail = ({ label, value, fullWidth, className }) => {
+const Detail = ({ label, value, fullWidth, className, dataCy = false }) => {
```

Comment:
probably don't need to define a default value / might as well just make this `null` since that's kind of what you're doing anyway when defining `labelCy` and `valueCy`